src/libsync/propagatedownload.h

```diff
@@ -15,33 +15,124 @@
 
 #include "owncloudpropagator.h"
 #include "networkjobs.h"
+#include "propagatecommonzsync.h"
 
 #include <QBuffer>
 #include <QFile>
 
 namespace OCC {
 
+class GETJob : public AbstractNetworkJob
```

Comment:
If you've got time, you could try tomake a separate commit that refactor GETJob and GETFileJob.
That'd make reviewer slightly easier.

Reply:
Based on the amount of times and nit-picking happening, my guess it oC don't really want to merge this. That's fine, I can withdraw my PRs and just see if nextCloud want to take them for free.

Reply:
My time is all up on this work now, it's taken way to long to get this far ...